patternfly/patternfly-3
# build(package): Remove extra files from the npm distribution

## Description

Currently we are distributing a bunch of extra files with our
npm package. This has inflated the size of the package being
installed, and this commit is to optimize that. More about
the solution can be read here:

https://docs.npmjs.com/files/package.json#files

To test:

- run `npm pack`
- run `tar -tf patternfly-<version>.tgz`
- verify the folder that was extracted only has the dist folder, package.json, LICENSE, and README.md

Related to JIRA ticket:

https://patternfly.atlassian.net/browse/OSUX-233

Related to GitHub issues:

https://github.com/patternfly/patternfly/issues/661
